lib/chef/knife/core/status_presenter.rb

```diff
@@ -101,6 +101,11 @@ def summarize(list)
             fqdn = (node[:ec2] && node[:ec2][:public_hostname]) || node[:fqdn]
             name = node["name"] || node.name
 
+            if !node["ohai_time"]
+              ui.error("Ohai has not yet ran on node #{name}.")
+              exit(1)
```

Comment:
This shouldn't cause the status list to exit; instead we should figure out a way of formatting the message to say that the node has never converged and keep going.

Reply:
Hi @thommay,

So perhaps we could use `ui.info()` instead of `ui.error()` to notify the user that the node has not yet converged, and then proceed to output the `knife status` message with a "time-since-last-run" value of 0.